### PR TITLE
feat: adds activity to admin management

### LINKED ISSUE #1094: Adding activity management to the Django Admin

### PR DESCRIPTION
**Pull request by yokwejuste**
Closes #1094
image

### CODE REVIEW

**Comment by NdibeRaymond** [comment]
we still need to rebase this right @yokwejuste ?

File: zubhub_backend/zubhub/activities/admin.py [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+    This function is used to unpublish selected activities
+    """
+    queryset.update(publish=False)
+    messages.success(request, "Selected records were unpublished successfully.")
+
+
+make_unpublished.short_description = "Unpublish selected activities"
+
+
+def make_published(modeladmin, request, queryset):
```
Comment (NdibeRaymond) [inline zubhub_backend/zubhub/activities/admin.py]: we can just name this `publish`

File: zubhub_backend/zubhub/activities/admin.py [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+    """
+    queryset = queryset.filter(publish=False)
+    queryset.update(publish=True)
+    messages.success(request, "Selected records were published successfully.")
+
+
+make_published.short_description = "Publish selected activities"
+
+
+def delete_selected(modeladmin, request, queryset):
```
Comment (NdibeRaymond) [inline zubhub_backend/zubhub/activities/admin.py]: a bit confused. Doesn't django admin do this automatically by itself already?

File: zubhub_backend/zubhub/activities/admin.py [hunk 3]
```diff
@@ -1,3 +1,49 @@
 from django.contrib import admin
+from django.contrib import messages
 
-# Register your models here.
+from .models import Activity
+
+
+def make_unpublished(modeladmin, request, queryset):
```
Comment (NdibeRaymond) [inline zubhub_backend/zubhub/activities/admin.py]: rename to `unpublish`

**Review by NdibeRaymond — CHANGES_REQUESTED** [review]

File: zubhub_backend/zubhub/activities/admin.py (hunk 2 above)
Comment (yokwejuste) [reply zubhub_backend/zubhub/activities/admin.py]: Not really, it doesn't do this. Reason why I added custom actions

File: zubhub_backend/zubhub/activities/admin.py (hunk 1 above)
Comment (yokwejuste) [reply zubhub_backend/zubhub/activities/admin.py]: Sure

File: zubhub_backend/zubhub/activities/admin.py (hunk 3 above)
Comment (yokwejuste) [reply zubhub_backend/zubhub/activities/admin.py]: Okay

**Pull request merged**